### PR TITLE
[MIG][9.0] contract_journal

### PR DESCRIPTION
**Pull request by lasley**
* Bump versions
* Update readme template
* Rename view and use odoo tags
* Remove deprecated yml
* Add tests
* Update invoice vals generate for v9 module

I'll handle v10 as well - that's really the version I need it on, just seemed like a dick move to skip a version in the migrations 😛

### CODE REVIEW

File: contract_journal/tests/test_contract_journal.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        """ It should inject the journal into invoice vals """
+        res = self.contract._prepare_invoice()
+        self.assertEqual(res['journal_id'], self.journal.id)
+
+    def test_default_journal_company(self):
+        """ It should return journal of correct company """
+        res = self.env['acount.analytic.account']._default_journal_id()
+        self.assertEqual(res.company_id, self.env.user.company_id)
+
+    def test_default_journal_type(self):
```
Comment (pedrobaeza) [inline contract_journal/tests/test_contract_journal.py:36]: Join this test with previous one

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Check Travis result

File: contract_journal/views/contract.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="account_analytic_account_journal_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.journal.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='template_id']" position="before">
```
Comment (pedrobaeza) [inline contract_journal/views/contract.xml:9]: The error is here

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: contract_journal/views/contract.xml (hunk 2 above)
Comment (lasley) [reply contract_journal/views/contract.xml:9]: Admittedly I should be testing my 9.0 submissions better, but mannnnn switching over to the v9 instance from the v10 in my local is a real pain 😦 . I swear one of these days I'm going to create us some sort of `odooenv`

**Comment by lasley** [comment]
I'm dumb though, I just found this in the new contract. I guess I've been looking at it the whole time; derp

**Pull request closed without merging**